src/EditorFeatures/CSharp/SplitStringLiteral/SplitStringLiteralCommandHandler.StringSplitter.cs

```diff
@@ -10,24 +10,23 @@
 namespace Microsoft.CodeAnalysis.Editor.CSharp.SplitStringLiteral
 {
     using Microsoft.CodeAnalysis.Indentation;
+    using Microsoft.CodeAnalysis.Options;
 
     internal partial class SplitStringLiteralCommandHandler
     {
         private abstract class StringSplitter
         {
             protected static readonly SyntaxAnnotation RightNodeAnnotation = new SyntaxAnnotation();
 
-            protected static readonly SyntaxToken PlusNewLineToken = SyntaxFactory.Token(
-                leading: default,
-                SyntaxKind.PlusToken,
-                SyntaxFactory.TriviaList(SyntaxFactory.ElasticCarriageReturnLineFeed));
+            protected readonly SyntaxToken PlusNewLineToken;
 
             protected readonly Document Document;
             protected readonly int CursorPosition;
             protected readonly SourceText SourceText;
             protected readonly SyntaxNode Root;
             protected readonly int TabSize;
             protected readonly bool UseTabs;
+            protected readonly SyntaxTriviaList NewLineTrivia;
```

Comment:
this field seems to be unnecessary (i can't see any reads of it anywhere).

Reply:
I’ve removed it.